src/main/java/com/session/service/client/SessionClientImpl.java

```diff
@@ -159,4 +135,22 @@ public boolean clear() throws SessionClientException {
             throw new SessionClientException("error reading content from http response", ex);
         }
     }
+
+    private Session getSession(String string) {
```

Comment:
I wonder if we should call this something like `sessionIdentifier`, or `sessionID` to be more descriptive?

Reply:
I have pushed an update and used `sessionIdentifier`. Great suggestion, needed something fairly generic.